packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -100,6 +108,74 @@ export default decorate([
           value={value.nbNodes}
         />
       </FormGrid.Row>
+      <FormGrid.Row>
+        <label>
+          {_('recipeStaticIpAddresses')}
+          <input
+            className='form-control'
+            name='staticIpAddress'
+            onChange={effects.toggleStaticIpAddress}
+            type='checkbox'
+            value={value.staticIpAddress}
+          />
+        </label>
+      </FormGrid.Row>
+      {value.staticIpAddress && [
+        <FormGrid.Row key='masterIpAddrRow'>
+          <label>{_('recipeMasterIpAddress')}</label>
+          <input
+            className='form-control'
+            name='masterIpAddress'
+            onChange={effects.onChangeValue}
+            placeholder={formatMessage(messages.recipeMasterIpAddress)}
+            required
+            type='text'
+            value={value.masterIpAddress}
+          />
+        </FormGrid.Row>,
+        <FormGrid.Row key='netmaskRow'>
+          <label>{_('recipeNetworkMask')}</label>
+          <input
+            className='form-control'
+            name='networkMask'
+            onChange={effects.onChangeValue}
+            placeholder={formatMessage(messages.recipeNetworkMask)}
+            required
+            type='text'
+            value={value.networkMask}
+          />
+        </FormGrid.Row>,
+        <FormGrid.Row key='gatewayRow'>
+          <label>{_('recipeGatewayIpAddress')}</label>
+          <input
+            className='form-control'
+            name='gatewayIpAddress'
+            onChange={effects.onChangeValue}
+            placeholder={formatMessage(messages.recipeGatewayIpAddress)}
+            required
+            type='text'
+            value={value.gatewayIpAddress}
+          />
+        </FormGrid.Row>,
+      ]}
+      {value.staticIpAddress &&
+        [...Array(+value.nbNodes)].map((v, i) => {
+          const workerIpAddress = 'workerIpAddress' + (i + 1)
+          return (
+            <FormGrid.Row key={i}>
```

Comment:
It's better to avoid using index value for key
 [See doc](https://reactjs.org/docs/lists-and-keys.html#keys)